app/handler/provider_complaint.py

```diff
@@ -151,7 +151,7 @@ def report_complaint_to_user_in_reply_phase(
     send_email_with_rate_control(
         alias.user,
         f"{ALERT_COMPLAINT_REPLY_PHASE}_{origin.name()}",
-        alias.user.email,
+        alias.mailbox.email,
```

Comment:
A reply can be sent either from one of the alias's mailboxes or an "authorized address".

Reply:
We don't know which authorized mbox sent it afaik.

Reply:
Just forwarded you an example of a complaint email. I noticed the `Return-Path` field that is the original envelope from and that should contains info about the email log that allows us to get the mailbox.

Reply:
Ah, thats a good one. In #963 I fixed it by getting the info from the report attached by yahoo that contains all the info or by extracting it from the `X-simplelogin-envelope-to`.